workshops/payments/__init__.py

```diff
@@ -0,0 +1,9 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017-2021, QIIME 2 development team.
```

Comment:
This doesn't match the "2016-2021" in the LICENSE - please update this (and all other files with "2017-2021") to match the license file.